examples/basic/gpu_shared.c

```diff
@@ -0,0 +1,191 @@
+/*
```

Comment:
You could rename this file to `gpu_shared_memory.c` as you have it in the CMakeLists.txt file.

Reply:
done